src/context/GurbLoadingContext.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState, createContext } from 'react'
+
+const GurbLoadingContext = createContext()
+
+export const GurbLoadingContextProvider = ({ children }) => {
+  const [loading, setLoading] = useState(undefined)
+
+  return (
+    <GurbLoadingContext.Provider
+      value={{
+        loading: loading,
+        setLoading: setLoading,
+      }}>
+      {children}
+    </GurbLoadingContext.Provider>
+  )
+}
+
+export default GurbLoadingContext
```

Comment:
I think Gurb management is simple enough to put everything in one context file. (it's just my opinion, this way works fine for me too)